### DISCUSSION TITLE
Feature request: Send speakers new feedback by mail

### DISCUSSION THREAD

**Discussion by BarbarossaTM** [pretalx/pretalx#1064] [Ideas]
## Problem you are facing

This is a nice-to-have feature request from a speakers point of view. I think it would be really cool if speakers would be informed by mail when feedback for their events has been provided. Currently speakers have to look into the system every now and then and check if someone left feedback.

## Possible Solution

Maybe allow the conference organized to active sending one mail per feedback submission or - if somehow easily possible - on mail per day or whatever time frame with new feedback or something like "there is new feedback, please check <here>"

Thank you for the awesome system and all your work <3

**Comment by rixx** [comment]
Agreed, this would be a cool thing to have. I can't prioritise it right now, but I'll happily help anybody who wants to implement it.

Notes for the implementation:
- Every speaker needs to be able to opt in. At first, a general opt-in is fine, but we should target a proposal-specific opt-in.
- While a limit like "at most one mail per day" sounds nice, I think at first we should go for "one mail until this person has seen the/any feedback".
